### ISSUE TITLE
save network to file

### ISSUE THREAD

**Issue by vasilibrutsky** [andersbll/neural_artistic_style#23]
hello! will your tool support saving the neural network to file? that I could import it each time and it would be much faster to process different images with the same effect

**Comment by andersbll** [comment]
Hi. You can use [pickle](https://docs.python.org/2/library/pickle.html) to save networks. :)

**Comment by vasilibrutsky** [comment]
Yes, but it would be perfect to extend functionality with key like 
python neural_artistic_style.py --subject images/tuebingen.jpg --style images/starry_night.jpg --save neuralnetwork.txt

and add like python neural_artistic_style.py --subject images/tuebingen.jpg --style images/starry_night.jpg --import neuralnetwork.txt

**Comment by andersbll** [comment]
If I have understood you correctly, you want to reuse a drawn image to draw other images faster. This is not possible as the algorithm starts drawing each image from scratch (from subject/noise).

Possibly related: You can specify an initial image as starting point for the optimization using the [--init](https://github.com/andersbll/neural_artistic_style/blob/master/neural_artistic_style.py#L67) parameter.
